src/main/java/net/sf/jsqlparser/util/deparser/DeleteDeParser.java

```diff
@@ -92,7 +92,11 @@ public void deParse(Delete delete) {
 
         if (delete.getWhere() != null) {
             buffer.append(" WHERE ");
-            delete.getWhere().accept(expressionVisitor);
+            int len = buffer.length();
```

Comment:
Please remove those specific parts from the PR as it does not fit into the generic Deparser.